internal/service/domain/sensor/sensor_test.go

```diff
@@ -0,0 +1,66 @@
+package sensor
+
+import (
+	"context"
+	"testing"
+
+	sensorUtils "github.com/green-ecolution/green-ecolution-backend/internal/server/http/handler/v1/sensor"
```

Comment:
Maybe this should be a separate package `sensorUtils` or `sensortest` (like `httptest` in standard lib)?

Reply:
We have those files also in the other folders. Maybe we can store the test data all in one file? I don't know what is the possible best way to handle those.

Reply:
👍